### DISCUSSION TITLE
What is the relationship between Elegosoft and CM3?

### DISCUSSION THREAD

**Discussion by nsgnkhibdk2cls0f** [modula3/cm3#1134] [General]
CM3 is owned by Elegosoft?

**Comment by olaf-wagner** [comment]
On 6 Jan 2023, at 20:37, nsgnkhibdk2cls0f wrote:

> CM3 is owned by Elegosoft?

No.

We just took over the code from Critical Mass after they stopped their own
business and published it as open source. We’ve also ported it to several
platforms many years ago, set up continuous integration and testing with
Tinderbox and Hudson and rolled out some releases. That’s all very long ago.

We haven’t contributed actively to the development of Modula-3 in recent
years, but try to keep the old resources available for those using them.

Olaf
-- 
Olaf Wagner -- elego Software Solutions GmbH -- http://www.elegosoft.com
               Gustav-Meyer-Allee 25 / Gebäude 12, 13355 Berlin, Germany
Geschäftsführer: Olaf Wagner | Sitz: Berlin
Handelregister: Amtsgericht Charlottenburg HRB 77719 | USt-IdNr: DE163214194

**Reply by ghost** [reply]
Do you use CM3 for your own products or you are just a good guy to pick CM3 up from Critical Mass and maintain it?

**Reply by nsgnkhibdk2cls0f** [reply]
@olaf-wagner Did you use CM3 for your own products? @tn1997tn's question is reasonable.

**Comment by olaf-wagner** [comment]
On 8 Jan 2023, at 19:02, nsgnkhibdk2cls0f wrote:

> @olaf-wagner Did you use CM3 for your own products? @tn1997tn's 
> question is reasonable.

We did, but that was long ago now, too.
I’m not sure if any of these are real questions or just trolls, and I 
don’t want
to feed them.

> -- 
> Reply to this email directly or view it on GitHub:
> https://github.com/modula3/cm3/discussions/1134#discussioncomment-4626776
> You are receiving this because you were mentioned.
>
> Message ID: 
> ***@***.***>
-- 
Olaf Wagner -- elego Software Solutions GmbH -- http://www.elegosoft.com
                Gustav-Meyer-Allee 25 / Gebäude 12, 13355 Berlin, 
Germany
Geschäftsführer: Olaf Wagner | Sitz: Berlin
Handelregister: Amtsgericht Charlottenburg HRB 77719 | USt-IdNr: 
DE163214194

**Reply by ghost** [reply]
This means you are just a good guy maintaining something you no longer use. What about fully transfer the documents to Github? Most of the information is still on https://modula3.elegosoft.com/cm3/ Your site used to be disappeared: https://github.com/modula3/cm3/discussions/1133 If you don't sell CM3 or wanted to sell CM3 in the future, why do you keep the documents only on your server?

**Reply by nsgnkhibdk2cls0f** [reply]
This is a real question, not trolls. Thank you for your answer. May I ask you that the product you mentioned is still being sold or still being maintained? I know you used simple past tense but I think I should ask to be more sure. Thank you again.

**Comment by olaf-wagner** [comment]
On 8 Jan 2023, at 22:04, nsgnkhibdk2cls0f wrote:

> This is a real question, not trolls. Thank you for your answer. May I 
> ask you that the product you mentioned is still being sold or still 
> being maintained? I know you used simple past tense but I think I 
> should ask to be more sure. Thank you again.

The product is not sold anymore; it has been obsolete for many years.
We currently don’t use Modula-3 for software development at Elego, as 
all
the base products we use, adapt and extend use other languages.

Regretfully the quality of a language has no implications on its 
popularity
or user base.

> -- 
> Reply to this email directly or view it on GitHub:
> https://github.com/modula3/cm3/discussions/1134#discussioncomment-4627591
> You are receiving this because you were mentioned.
>
> Message ID: 
> ***@***.***>
-- 
Olaf Wagner -- elego Software Solutions GmbH -- http://www.elegosoft.com
                Gustav-Meyer-Allee 25 / Gebäude 12, 13355 Berlin, 
Germany
phone: +49 30 23 45 86 96  mobile: +49 177 2345 869  fax: +49 30 23 45 
86 95
Geschäftsführer: Olaf Wagner | Sitz: Berlin
Handelregister: Amtsgericht Charlottenburg HRB 77719 | USt-IdNr: 
DE163214194
